python/tk_multi_publish2/api/task.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2018 Shotgun Software Inc.
```

Comment:
New version of this: https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/python/tk_multi_publish2/processing/task.py

Internals rewired a bit.